docs/src/prototypes/IconViewer/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react'
+import { Provider, Grid, Divider, Header, Icon } from '@stardust-ui/react'
+import { processedIconsTheme } from 'src/themes/teams/iconTest'
+
+const cellStyles = {
+  margin: '10px 0',
+}
+
+class IconViewerExample extends React.Component<any, {}> {
+  render() {
+    return (
+      <div style={{ margin: '20px' }}>
+        <Header
+          as="h3"
+          content="Teams Icons viewer"
+          description={{
+            content:
+              'These icons have been pulled directly from the Angular app and are ready to be added to the Teams theme in Stardust as needed',
+            styles: { fontSize: '16px' },
+          }}
+        />
+        <Provider theme={processedIconsTheme}>
+          <Provider.Consumer
+            render={theme => (
+              <div>
+                <div>
+                  <Divider>
+                    <Header as="h3" content="Regular" />
+                  </Divider>
+                  <Grid columns={4} style={{ textAlign: 'center' }}>
+                    {Object.keys(theme.icons)
+                      .filter(name => name.indexOf('processedIcons_') !== -1)
```

Comment:
would suggest to declare constant, like

```js
const processedIconsNamePrefix = 'processedIcons_'
```

and reuse it, just to reduce duplication of this string literal

Reply:
it would be great to ensure that icons are sorted alphabetically, like

```js
const processedIconNamePrefix = 'processedIcons_'

Object.keys(theme.icons)
  .filter(name => name.startsWith(processedIconNamePrefix))   // <-- 'startsWith' would be more expressive 
  .map(name => name.replace(processedIconNamePrefix, ''))
  .sort()
  .map(name => <div>...</div>)